Maple2.Server.Game/PacketHandlers/MapleopolyHandler.cs

```diff
@@ -85,33 +80,32 @@ private enum Command : byte {
         int dice1 = Random.Shared.Next(1, 7);
         int dice2 = Random.Shared.Next(1, 7);
 
-        totalTileValue += dice1 + dice2;
+        totalSlotValue += dice1 + dice2;
         if (dice1 == dice2) {
             freeRollValue++;
         }
 
         session.GameEventUserValue.Set(gameEvent.Id, GameEventUserValueType.MapleopolyFreeRollAmount, freeRollValue);
-        session.GameEventUserValue.Set(gameEvent.Id, GameEventUserValueType.MapleopolyTotalTileCount, totalTileValue);
-        session.Send(MapleopolyPacket.Roll(totalTileValue, dice1, dice2));
+        session.GameEventUserValue.Set(gameEvent.Id, GameEventUserValueType.MapleopolyTotalSlotCount, totalSlotValue);
+        session.Send(MapleopolyPacket.Roll(totalSlotValue, dice1, dice2));
     }
 
     private void HandleResult(GameSession session) {
-        GameEvent? gameEvent = session.FindEvent<BlueMarble>();
-        if (gameEvent == null) {
+        GameEvent? gameEvent = session.FindEvent(GameEventType.BlueMarble);
+        if (gameEvent?.Metadata.Data is not BlueMarble blueMarble) {
```

Comment:
i could be wrong, but do you not need `gameEvent?.Metadata?.Data` rather than just `gameEvent?.Metadata.Data`?

(This same issue is in multiple places if you end up changing it here)

Reply:
Metadata on GameEvent is never null

Reply:
That's true but if you do null propagation, I swear it will still propagate the null to Metadata. Am I wrong here?

Reply:
Never mind, I'm wrong... I guess I'm way too rusty